python/core/looping/using-else-in-loops.md

```diff
@@ -26,7 +26,7 @@ standards:
 
 Unlike other programming languages such as C, Python allows the use of `else` statements with loops.
 
-They are implemented so once a `for` or `while` loops condition evaluates to false, the `else` is executed. In other words, the `else` statement is called **once** when loops exit naturally.
+They are implemented so that once a `for` or `while` loops condition evaluates to false, the `else` is executed. In other words, the `else` statement is called **once** when the loops exit naturally.
```

Comment:
what do you think about this?

```suggestion
They are implemented so that once a `for` or a `while` loop condition evaluates to false, the `else` is executed. In other words, the `else` statement is called **once** when the loops exit naturally.
```

Reply:
much better!